app/views/likes/_buttons.html.erb

```diff
@@ -0,0 +1,20 @@
+<% if logged_in? %>
+  <% if current_chef.likes.find {|like| like.recipe_id == object.id} %>
+    <% like = current_chef.likes.find {|like| like.recipe_id == object.id} %>
+    <%= link_to recipe_like_path(recipe_id: object.id, id: like.id), class: "dislike-button-#{object.id}", remote: true, method: :delete do %>
+      <button class="btn btn-sm active px-3 btn-outline-primary">
+        <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-hand-thumbs-up-fill" viewBox="0 0 16 16">
```

Comment:
definitely hard to read these files with the svg embedded in them... Would be nice to use an icon library.. but if we can't, we'll have to fix later...

Reply:
I agree, is fixed now!